share/spice/dota2/content.handlebars

```diff
@@ -0,0 +1,149 @@
+{{#if isItem}}
+    <div style="float: left; width: 100px;">
```

Comment:
Let's use an external stylesheet for all the CSS. You can create `/share/spice/dota2/dota2.css` and it will automatically be loaded for you.

You'll need to prepend all the CSS rules with `.zci--dota2` though, so that your CSS only targets the elements inside your IA's `<div>` -- we automatically create a wrapper div and give it a class with `zci--<ia_id>` for you.

----

Also, we have existing classes you can use to float left/right: `pull-right`, `pull-left`. Similarly we have classes and CSS pre-made to handle grid layouts that will work on mobile so it may be best to use those.

You can read about our other classes (font sizes, background colors, etc.) here: https://duckduckgo.com/styleguide